apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -133,6 +132,40 @@ const SwapForm = () => {
   const refreshTime = useRefreshRates(swapTransaction.swap, {
     pause: pauseRefreshing,
   });
+
+  const getCustomFeesPerFamily = useCallback(async transaction => {
```

Comment:
Love this function, I wonder would it be better placed in `ledger-live-common`? WDYT?

Reply:
I like it. Done!